data/human/fleets.txt

```diff
@@ -1837,6 +1843,24 @@ fleet "Small Southern Pirates"
 		"Clipper (Speedy)"
 	variant 1
 		"Modified Argosy"
+	variant 2
+		"Scrapper"
+	variant 2
+		"Scrapper (Minimal)"
```

Comment:
I don't think spawning 1 of them should be more likely than spawning two, looking at the rest of the variants they have at least 3 sparrows of strength each, this would significantly weaken these fleets (mb thats intended?)

I'd say spawn 2-3 of them, except in a few rare cases? especially if its a weak version of it I can see pirates being stupid enough to attack with it but not alone

they wont rush towards the player anyway because they consider cost (not heroic) and there is no way you cost less than they do, right?

that said, feel free to ignore me this is content, so not my jam